### ISSUE TITLE
[Bug] creepjs detects stealth extensions

### ISSUE THREAD

**Issue by searchingforcode** [berstend/puppeteer-extra#702]
![image](https://user-images.githubusercontent.com/1987330/187315575-aab06cb6-50c7-45a4-8a5a-51bf3ede1ed2.png)

Hello creepjs fully detects it and even can track on each visit if i use puppeteer stealth. It also uses the service worker or something like that which leaks all real info to creepjs. Any plan on this? 

Thanks

**Comment by Bllacky** [comment]
I just used creep.js on my laptop's Firefox browser, giving me a score of -F. I have my doubts it works correctly.

**Comment by searchingforcode** [comment]
> I just used creep.js on my laptop's Firefox browser, giving me a score of -F. I have my doubts it works correctly. My Chrome browser got C+ and Brave F-. Not sure what creep.js detects, but they will block many users.

Yeah they are very aggressive its true but their workers based detection system can detect all real info.

**Comment by SinicalPyx** [comment]
It's all in creeps code

`if (prototypeLiesLen >= 13 &&
					puppeteerExtra.contentDocumentHash.includes(hash.contentDocumentHash) &&
					puppeteerExtra.contentWindowHash.includes(hash.contentWindowHash) &&
					puppeteerExtra.createElementHash.includes(hash.createElementHash) &&
					puppeteerExtra.getElementByIdHash.includes(hash.getElementByIdHash) &&
					puppeteerExtra.appendHash.includes(hash.appendHash) &&
					puppeteerExtra.insertAdjacentElementHash.includes(hash.insertAdjacentElementHash) &&
					puppeteerExtra.insertAdjacentHTMLHash.includes(hash.insertAdjacentHTMLHash) &&
					puppeteerExtra.insertAdjacentTextHash.includes(hash.insertAdjacentTextHash) &&
					puppeteerExtra.prependHash.includes(hash.prependHash) &&
					puppeteerExtra.replaceWithHash.includes(hash.replaceWithHash) &&
					puppeteerExtra.appendChildHash.includes(hash.appendChildHash) &&
					puppeteerExtra.insertBeforeHash.includes(hash.insertBeforeHash) &&
					puppeteerExtra.contentDocumentHash.includes(hash.contentDocumentHash) &&
					puppeteerExtra.replaceChildHash.includes(hash.replaceChildHash) &&
					puppeteerExtra.getContextHash.includes(hash.getContextHash) &&
					puppeteerExtra.toDataURLHash.includes(hash.toDataURLHash) &&
					puppeteerExtra.toBlobHash.includes(hash.toBlobHash) &&
					puppeteerExtra.getImageDataHash.includes(hash.getImageDataHash) &&
					puppeteerExtra.hardwareConcurrencyHash.includes(hash.hardwareConcurrencyHash)) {
					return 'puppeteer-extra'
				}`

**Comment by drewhoener** [comment]
Bumping this with more creepjs detection
![image](https://user-images.githubusercontent.com/6218989/208003906-bbf61ac1-4ed1-454f-8ea3-46cfd105791d.png)
![image](https://user-images.githubusercontent.com/6218989/208003941-561e5a27-42a0-469b-ba68-004a3e2c5919.png)

![image](https://user-images.githubusercontent.com/6218989/208003990-becf5bff-8fd8-4738-b24a-bbfa7f787bc4.png)
